src/App.tsx

```diff
@@ -0,0 +1,194 @@
+import {
+  AppBar,
+  Alert,
+  Box,
+  Button,
+  Container,
+  FormGroup,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+  Toolbar,
+  Typography,
+} from "@mui/material";
+import SendIcon from "@mui/icons-material/Send";
+import React from "react";
+import CodeMirror from "@uiw/react-codemirror";
+import { oneDark } from "@codemirror/theme-one-dark";
+import init, { program } from "miden-wasm";
+import "./App.css";
+
+async function getExample(example: string[], inputs: Boolean) {
+  if (inputs) {
+    const response = fetch(`https://raw.githubusercontent.com/0xPolygonMiden/examples/main/examples/${example}.inputs`)
+    return (await response).text();
+  }
+  else {
+    const response = fetch(`https://raw.githubusercontent.com/0xPolygonMiden/examples/main/examples/${example}.masm`)
+    return (await response).text();
+  }
+}
```

Comment:
We should probably return an object with inputs and code instead of calling the function twice and taking inputs boolean flag.